skywalking-sniffer/skywalking-toolkit-activation/skywalking-toolkit-opentracing-activation/src/main/java/com/a/eye/skywalking/toolkit/activation/opentracing/spanbuilder/OpenTracingLocalBuriedPointType.java

```diff
@@ -0,0 +1,27 @@
+package com.a.eye.skywalking.toolkit.activation.opentracing.spanbuilder;
+
+import com.a.eye.skywalking.api.IBuriedPointType;
+
+/**
+ * Created by xin on 2017/1/16.
+ */
+public class OpenTracingLocalBuriedPointType implements IBuriedPointType {
```

Comment:
Only a suggestion, refactor class type to `enum` seems better. Also suggest in other plugins.